src/components/Icons/WalletConnectIcon.tsx

```diff
@@ -17,3 +17,16 @@ export const WalletConnectIcon = createIcon({
   ),
   viewBox: '0 0 300 185',
 })
+
+export const WalletConnectCurrentColorIcon = createIcon({
+  displayName: 'WalletConnectIcon',
```

Comment:
This has the same displayName as the existing `WalletConnectIcon`, let's keep it consistent with the const name